web/templates/settings/notification_settings.hbs

```diff
@@ -8,22 +8,27 @@
         <table class="notification-table table table-bordered wrapped-table">
             <thead>
                 <tr>
-                    <th rowspan="2" width="30%"></th>
-                    <th colspan="2" width="28%">{{t "Desktop"}}</th>
+                    <th rowspan="2" width="30%"><div class="notification-table-header-content"></div></th>
```

Comment:
I don't understand the inclusion of this element everywhere, especially when it's empty.